core/modules/node/node.module

```diff
@@ -856,15 +856,15 @@ function node_invoke($node, $hook, $a2 = NULL, $a3 = NULL, $a4 = NULL) {
  * from the database. Nodes are loaded into memory and will not require database
  * access if loaded again during the same page request.
  *
- * @param $nids
- *   (optional) An array of node IDs.
- * @param $conditions
+ * @param array|false $nids
+ *   (optional) An array of node IDs or FALSE to load all the nodes.
```

Comment:
It might also be worth noting what an empty array does... I believe @argiepiano's PR addressed this.

Reply:
When an empty array is passed as first argument to [`node_load_multiple()`](https://docs.backdropcms.org/api/backdrop/core%21modules%21node%21node.module/function/node_load_multiple/1), the function returns an empty array. At least, that is what I understand from [`DefaultEntityController::load()`](https://docs.backdropcms.org/api/backdrop/core%21modules%21entity%21entity.controller.inc/function/DefaultEntityController%3A%3Aload/1), where the returned value (`$entities`) is initialized to an empty array.  
Is that correct?

Reply:
The return value is type-hinted with `Node[]`. Does that include an empty array as I am thinking it does?

Reply:
Yes, the return value is an array and, technically, doesn't need to be documented. Still, I think it would be helpful to mention it. I've been "bitten" by this behavior: the default value for the 2nd parameter in `entity_load_multiple('node')` returns all nodes, but the default value for `node_load_multiple()` returns an empty array. It's a bit inconsistent.

But if you feel strongly about not mentioning it, that's fine.

Reply:
@argiepiano I am not contrary to mention that. I was trying to understand what the exact `node_load_multiple()` behavior is (in the case I misunderstood it) and what exactly should be documented (which is what your last comment made me understand).